src/models/generated/EconThermal.jl

```diff
@@ -0,0 +1,50 @@
+#=
+This file is auto-generated. Do not edit.
+=#
+
+"""Data Structure for the economical parameters of thermal generation technologies."""
+mutable struct EconThermal <: PowerSystems.TechnicalParams
+    capacity::Float64
+    variablecost::Union{Tuple{Float64, Float64}, Array{Tuple{Float64, Float64}, N} where N}
+    fixedcost::Float64
+    startupcost::Float64
+    shutdncost::Float64
+    annualcapacityfactor::Union{Nothing, Float64}
```

Comment:
This field should be removed. AnnucalCapacityFactor isn't relevant for operations.

Reply:
Please file an issue.  This requires changes in the parsing code and is not relevant to this PR.

Reply:
Ok, I will open an issue to do an overall review of all the structs.